internals/secrethub/signup.go

```diff
@@ -181,7 +185,37 @@ func (cmd *SignUpCommand) Run() error {
 	}
 
 	cmd.progressPrinter.Stop()
-
 	fmt.Fprintf(cmd.io.Stdout(), "Setup complete. To read your first secret, run:\n\n    secrethub read %s\n\n", secretPath)
```

Comment:
I think this line should be printed after the entire thing is complete. now it gets lost in translation. so the flow becomes:

1. questions for account
2. questions for org
3. create account
4. create org
5. success message / helpful error message if org already exists

Reply:
We discussed last Friday that separating the creation of the account and the org workspace gives us two advantages:
1. It keeps the signup part simple and straightforward and adds the shared workspace as an optional extra goodie. We don't want to create extra hurdles in signing up.
1. It makes messages easier and clearer, as they're given in the context of the phase they're printed in. "workspace already exists" in the account creation phase means the username already exists, in the shared workspace creation phase it means the orgname already exists.


To solve the problem of the "to read your first secret ..." being printed before the command has finished. How about printing "Setup complete." where it is right now, but move the "to read your first secret ..." to the end of the command?

Reply:
I feel we mean the same thing but it looks like we're disagreeing. Lets discuss this in person, e.g. tomorrow.